src/tests/ftest/daos_test/DaosCoreTest.py

```diff
@@ -80,7 +81,7 @@ def tearDown(self):
                     subprocess.check_call(['ssh', host,
                                            '[ -f \"{0}\" ] && '
                                            '    mv \"{0}\" '
-                                           '    \"{1}\"'.format(logfile,
+                                           '    \"{1}\"'.format("/tmp/daos.log",
```

Comment:
Default path is /tmp/daos.log and for  CI it will copy that file in to /tmp/Functional_$TEST_TAG/.

Reply:
If we export D_LOG_FILE=/tmp/Functional_$TEST_TAG/daos.log, client and server log will be in the same file. How this are getting separated in CI?

Reply:
I am not sure how it worked before as I do not find client side log. With this PR client and sever logs are getting copied in same file.
For example: https://build.hpdd.intel.com/job/daos-stack/job/daos/view/change-requests/job/PR-226/35/artifact/Functional/IO%20test_daos.log.wolf-103vm9/*view*/ 

04/01-20:38:43.14 wolf-103vm9 DAOS[11118] **client** DBUG src/common/tse.c:350 register_cb() alloc(calloc) 'dtc': 40 at 0x7f976e6520b0.
04/01-20:38:43.14 wolf-103vm9 DAOS[11118] object DBUG src/object/cli_obj.c:152 obj_alloc() alloc(calloc) 'obj': 160 at 0x7f976e652fb0.

Reply:
> I am not sure how it worked before

The way it worked before is that the log path was set in the environment when starting the servers so that they logged to `/tmp/Functional_$TEST_TAG/daos.log` and the clients were left to log to the default path, `/tmp/daos.log`,

The key here was telling the server to log to a different file, not simply renaming the default file after the test was complete.

Reply:
This was not mention in log document so difficult to understand the behavior. So we need client log and after talking with Mohamad we can get that by passing the orterun -x D_LOG_FILE=/tmp/client.log for daos_test. So I am thinking to update the command and use that as client log

Reply:
I can see separate client log with this change.